README.md

```diff
@@ -1,46 +1,54 @@
+A [Visual Studio Code](https://code.visualstudio.com) (VSCode) extension to help working with [IES](https://github.com/dstl/IES4) in [Terse RDF Triple Language (TTL)](https://www.w3.org/TR/turtle/) (.ttl) files
+
+Features:
+<ul>
+  <li>
+    <strong>Colour highlighting</strong> - Highlights IES classes according to their defined colours<br>
+    <img src="./images/Overview.png" alt="Highlights IES classes according to their defined colours" style="height:300px">
+  </li>
+  <li>
+    <strong>Diagnostics</strong> - Syntax highlights missing commas, dots, semicolons<br>
+    <img src="./images/Diagnostics.png" alt="Syntax highlights missing commas, dots, semicolons">
+  </li>
+  <li>
+    <strong>IES class suggestions</strong> - Suggests IES classes as you type<br>
+    <img src="./images/Suggestions.jpeg" alt="Suggests IES classes as you type" style="height:300px">
+  </li>
+  <li>
+    <strong>On hover matches</strong> - Hover over a URI to reveal IES type<br>
+    <img src="./images/OnHover.png" alt="Hover over a URI to reveal IES type">
+  </li>
+</ul>
 
-More information about IES can be found here: [IES4 GitHub repository](https://github.com/dstl/IES4)
 
-### Features
 
-- **Colour highlighting:** Highlights IES classes according to their defined colours
-- **Diagnostics:** Syntax highlights missing commas, dots, semicolons
-- **Autocomplete Classes:** Suggests IES classes as you type
-- **On-hover reveal URI type** Hover-over a URI to reveal its type
 
-#### Color highlighting
-![colour highlighting screenshot](./images/Overview.png)
-#### Diagnostics
-![syntax diagnostics screenshot](./images/Diagnostics.png)
-#### IES class suggestions
-![IES class suggestion screenshot](./images/Suggestions.jpeg)
-#### On hover matches
-![On-hover IES type tracking](./images/OnHover.png)
 
 ## Install
 
-1. Download the latest release from: [IES Syntax Highlighter releases](https://github.com/Telicent-io/ies-ext/releases)
+1. Download latest [/releases](/releases)
+
+In VSCode:
 
-2. Bring up the Extensions view by clicking on the Extensions icon in the **Activity Bar** on the side of VS Code or the **View: Extensions** command `Cmd/Ctrl + Shift + x`.
+2. Click "Extensions" icon in the **Activity Bar** on the side of VS Code or the **View: Extensions** command `Cmd/Ctrl + Shift + x`.
 
 3. Click on the Extensions view command dropdown, the three dots (`...`), located at the top of the Extensions view and select **Install from VSIX...**
 
-4. Navigate to the downloaded file and select it.
+4. Navigate to the downloaded release file (step 1) and select it.
 
-5. Select theme IES Highlighter dark+ default `Cmd/Ctrl + k` then `Cmd/Ctrl + t`. Or go to the Extensions view -> ies-extension -> Set Color Theme.
+5. Select theme IES Highlighter dark+ default `Cmd/Ctrl + k` then `Cmd/Ctrl + t`. Or go to the Extensions view -> ies-extension -> Set Colour Theme.
```

Comment:
I don't know if it's worth mentioning here or somewhere else in the README.md that syntax highlighting is really only effective using a dark coloured background to the text.  For example syntax highlighting renders the 'ies:' prefix white.

See issue [Syntax Highlighting is only properly visible using a dark theme #2](https://github.com/Telicent-io/ies-syntax-highlighter-vscode-extension/issues/2) for more info.

Reply:
It's not a new issue: it was raised as a bug / issue on the internal version of the repository (i.e. telicent-io) but might still be present as an issue in this OSS version (I couldn't download a release to confirm, though).

The 'solution' might be as suggested in my review comment to just warn any users of a light theme (lucky me!) that syntax highlighting might provide a 'sub-optimal' user experience.

Reply:
Oh, I thought the issue was in oss. Ok i'll change

Reply:
I assume you'll just tweak the README.md.  

If you were planning to modify the colour-scheme or tweak the code I'd check in with Andreas or someone in the Data and Ontology Team, first.

This is because there is significance to the colours used in the syntax highlighter.

Reply:
Yeah just updated readme: [92a8957](https://github.com/telicent-oss/ies-syntax-highlighter-vscode-extension/pull/8/commits/92a8957a4e7eb45df75a02ee3f3318d2c4c8c3a9)